package.json

```diff
@@ -24,28 +24,36 @@
     "btoa": "^1.1.2",
     "form-urlencoded": "^1.2.0",
     "http-status": "^0.2.0",
-    "nock": "^7.5.0",
+    "nock": "^8.0.0",
     "node-fetch": "^1.3.3",
     "node-uuid": "^1.4.7",
     "q": "^1.4.1"
   },
   "devDependencies": {
+    "@graviton/graviton-typings": "1.0.0",
```

Comment:
@awaltert You are right, I created a ticket regarding that: https://github.com/zalando-incubator/lib-oauth-tooling/issues/25